protocol-designer/src/pages/Designer/ProtocolSteps/StepForm/StepTools/MixTools/index.tsx

```diff
@@ -1,3 +1,108 @@
-export function MixTools(): JSX.Element {
-  return <div>TODO: wire this up</div>
+import { useSelector } from 'react-redux'
+import { useTranslation } from 'react-i18next'
+import { DIRECTION_COLUMN, Divider, Flex } from '@opentrons/components'
+import { InputStepFormField } from '../../../../../../molecules'
+import {
+  getLabwareEntities,
+  getPipetteEntities,
+} from '../../../../../../step-forms/selectors'
+import { getEnableReturnTip } from '../../../../../../feature-flags/selectors'
+import {
+  ChangeTipField,
+  DropTipField,
+  LabwareField,
+  PartialTipField,
+  PickUpTipField,
+  PipetteField,
+  TipWellSelectionField,
+  TiprackField,
+  VolumeField,
+  WellSelectionField,
+} from '../../PipetteFields'
+import type { StepFormProps } from '../../types'
+
+export function MixTools(props: StepFormProps): JSX.Element {
+  const pipettes = useSelector(getPipetteEntities)
+  const enableReturnTip = useSelector(getEnableReturnTip)
+  const labwares = useSelector(getLabwareEntities)
+  const { t } = useTranslation(['application', 'form'])
+
+  const { propsForFields, formData, toolboxStep } = props
```

Comment:
to follow the other components

```suggestion
  const { propsForFields, formData, toolboxStep } = props
  const { t } = useTranslation(['application', 'form'])
  const pipettes = useSelector(getPipetteEntities)
  const enableReturnTip = useSelector(getEnableReturnTip)
  const labwares = useSelector(getLabwareEntities)
```